common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -541,16 +491,22 @@ func (v *pinotVisibilityTripleManager) CountWorkflowExecutions(
 		closeStatus: 6, // 6 means not set closeStatus.
 		customQuery: request.Query,
 	}, request.Domain)
-	manager := v.chooseVisibilityManagerForRead(request.Domain)
+	manager := v.chooseVisibilityManagerForRead(ctx, request.Domain)
+	return manager.CountWorkflowExecutions(ctx, request)
+}
+
+func (v *pinotVisibilityTripleManager) chooseVisibilityManagerForRead(ctx context.Context, domain string) VisibilityManager {
+	// temporary code: for Pinot migration usage -->
 	if override := ctx.Value(ContextKey); override == Primary {
-		manager = v.esVisibilityManager
+		v.logger.Debug("primary was chosen")
```

Comment:
let's change the message here to make it more reasonable to others, like primary visibility manager elasticsearch was chosen for read